### PR TITLE
Syntax list updates

### PR DESCRIPTION
**Pull request by switch-case**
Pulled syntax items into an array and made some style updates, feel free to ditch the styles, but I do think the grid updates is needed so they don't get squished so much on mobile.

### CODE REVIEW

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx [hunk 1]
```diff
@@ -28,123 +28,116 @@ const ComingSoon = () => {
...
       </Grid>
     </>
   );
 };
 
-export default ComingSoon;
+const SyntaxItems = [
+  {
+    title: "Comparisons",
+    content: `filter:> filter: >= filter:< filter<=`,
```
Comment (manwaring) [inline src/pages/Cards/CardSearch/Syntax/Syntax.tsx:53]: would it be simpler if the content was `> OR >= OR < OR <=` to match what you've updated on the other ones?

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx [hunk 2]
```diff
@@ -28,123 +28,116 @@ const ComingSoon = () => {
...
+    example: "power:>4"
+  },
+  {
+    title: "Exclusions",
+    content: `-filter: !filter:`,
```
Comment (manwaring) [inline src/pages/Cards/CardSearch/Syntax/Syntax.tsx:58]: same as above - `- OR !`

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx [hunk 3]
```diff
@@ -28,123 +28,116 @@ const ComingSoon = () => {
...
+    example: "i:<4"
+  },
+  {
+    title: "Keyword",
+    content: `keyword: OR k:`,
+    example: "k:crush+dominate"
+  },
+  {
+    title: "Life",
+    content: `life: OR l:`,
```
Comment (manwaring) [inline src/pages/Cards/CardSearch/Syntax/Syntax.tsx:98]: my bad, "l" won't work anymore, it's updated to "li" for the abbreviation

**Review by manwaring — APPROVED** [review]
I think Legal, And, Or, and Restricted Formats are missing - otherwise this looks amazing

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx [hunk 4]
```diff
@@ -28,123 +28,116 @@ const ComingSoon = () => {
...
+  },
+  {
+    title: "Talents",
+    content: `talent: OR t:`,
+    example: "t:shadow"
+  },
+  {
+    title: "Type",
+    content: `type: OR t:`,
+    example: "t:wizard"
```
Comment (manwaring) [inline src/pages/Cards/CardSearch/Syntax/Syntax.tsx:139]: wizard would go with class - type is things like action, hero, equipment

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx [hunk 5]
```diff
@@ -28,123 +29,114 @@ const ComingSoon = () => {
...
+  },
+  {
+    title: "Intellect",
+    content: `intellect: OR i:`,
+    example: "i:<4"
+  },
+  {
+    title: "Keyword",
+    content: `keyword: OR k:`,
+    example: "k:crush+dominate"
```
Comment (manwaring) [inline src/pages/Cards/CardSearch/Syntax/Syntax.tsx:93]: I don't think there's a card with Crush AND Dominate, is there?

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx [hunk 6]
```diff
@@ -28,123 +29,114 @@ const ComingSoon = () => {
...
+  },
+  {
+    title: "Life",
+    content: `life: OR li:`,
+    example: "l:<20"
```
Comment (manwaring) [inline src/pages/Cards/CardSearch/Syntax/Syntax.tsx]: my bad again, example also needs "li"

Comment (switch-case) [reply src/pages/Cards/CardSearch/Syntax/Syntax.tsx]: Yeah, caught that in my recent update

File: src/pages/Cards/CardSearch/Syntax/Syntax.tsx (hunk 5 above)
Comment (switch-case) [reply src/pages/Cards/CardSearch/Syntax/Syntax.tsx:103]: 🤷

Comment (switch-case) [reply src/pages/Cards/CardSearch/Syntax/Syntax.tsx:103]: One can hope

**Comment by switch-case** [comment]
> I think Legal, And, Or, and Restricted Formats are missing - otherwise this looks amazing

Added

**Review by manwaring — APPROVED** [review]

**Comment by manwaring** [comment]
fixes #21

**Comment by switch-case** [comment]
Syntax list structure and style updates

**Pull request merged**